manual/webpack/webpack.dev.js

```diff
@@ -6,14 +6,10 @@ const external = require.resolve('../../dist/scout')
 const URLImportPlugin = require("../../dist/webpack")
 const commonPaths = require('./paths');
 
-console.log(URLImportPlugin)
 module.exports = {
   mode: 'development',
   devtool: 'source-map',
-  entry: {
-    main: require.resolve('../src/index.jsx'),
-    other: require.resolve('../src/anoterEntry.js'),
-  },
+  entry: commonPaths[process.env.MFE],
```

Comment:
This is `undefined`

Reply:
ahh hold on, this should come from package json